docs/zh/recipes/svelte.md

```diff
@@ -68,14 +70,15 @@ The toggleService has a `.subscribe` function that is similar to Svelte stores,
 
 ```html
 <script>
-    import {interpret} from 'xstate';
-    import {toggleMachine} from './machine';
+  import { interpret } from 'xstate';
+  import { toggleMachine } from './machine';
 
-    const toggleService = interpret(toggleMachine).start();
+  const toggleService = interpret(toggleMachine).start();
 </script>
 
-<button on:click={() => toggleService.send({ type: 'TOGGLE' })}>
-    {$toggleService.matches('inactive') ? 'Off' : 'On'}
+<button on:click="{()" ="">
+  toggleService.send({ type: 'TOGGLE' })}> {$toggleService.matches('inactive') ?
+  'Off' : 'On'}
```

Comment:
Whoa, what happened here?

Reply:
Looks like an issue with the `html` tag on those code blocks. There's [a plugin for `svelte`](https://github.com/sveltejs/prettier-plugin-svelte) but I think it'd be easier to just add `docs` to `.prettierignore` since it's not really updated anymore. What do you think?

Reply:
Yes, can you please do that?